storage/experimental/experimental.go

```diff
@@ -25,9 +25,27 @@ import (
 	"time"
 
 	"cloud.google.com/go/storage/internal"
+	"go.opentelemetry.io/otel/sdk/metric"
 	"google.golang.org/api/option"
 )
 
+// WithMetricInterval provides a [ClientOption] that may be passed to [storage.NewGrpcClient].
+// It sets how often to emit metrics when using NewPeriodicReader and must be
+// greater than 1 minute.
+// https://pkg.go.dev/go.opentelemetry.io/otel/sdk/metric#NewPeriodicReader
```

Comment:
leave an empty line (with a `//` only) between links if you want them to not wrap in the godoc.